pkg/manager/member/tikv_upgrader.go

```diff
@@ -93,12 +93,19 @@ func (u *tikvUpgrader) Upgrade(meta metav1.Object, oldSet *apps.StatefulSet, new
 		return nil
 	}
 
+	// if TiKV not bootstrapped, we set partition to 0 to upgrade quickly
+	if !status.BootStrapped {
+		setUpgradePartition(newSet, 0)
+		return nil
+	}
+
```

Comment:
Is this necessary after the change of L108?

Reply:
Good suggestion, I will delelte this code.